### DISCUSSION TITLE
LettuceBasedProxyManagerBuilder withExpirationStrategy deprecated but what should be the substitute?

### DISCUSSION THREAD

**Discussion by MikeMJ** [vladimir-bukhtoyarov/bucket4j#473] [General]
I'm trying to get a LettuceBasedProxyManager with the following code. But it seems the withExpirationStrategy method has been deprecated. 
`		return LettuceBasedProxyManager.builderFor(redisConnection)
				.withExpirationStrategy(
						ExpirationAfterWriteStrategy.basedOnTimeForRefillingBucketUpToMax(Duration.ofMinutes(1L)))
				.build();`

Unfortunately, I wasn't able to find any documentation for what we should use instead. Can someone suggest what to use?

**Comment by vladimir-bukhtoyarov** [comment]
Hello @MikeMJ 

Please read this https://github.com/bucket4j/bucket4j/issues/471

**Reply by MikeMJ** [reply]
Thanks for that. I've changed the code to use the currently available option. Waiting for the 8.11.0.
